feature-pack/src/main/resources/modules/system/layers/base/org/jboss/weld/core/main/module.xml

```diff
@@ -45,8 +48,8 @@
         <module name="javax.validation.api"/>
         <module name="org.glassfish.javax.el" />
         <module name="org.jboss.classfilewriter"/>
-        <module name="org.jboss.weld.api" />
-        <module name="org.jboss.weld.spi" />
+        <module name="org.jboss.weld.api" export="true"/>
```

Comment:
Why have these been changed to be exported?

Reply:
The reason was this issue: https://issues.jboss.org/browse/WELD-1914. All proxies/subclasses now implement `WeldConstruct` from API. But even so, I think it makes sense that any `weld-core-impl` consumer also depends on weld API and SPI.